cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -56,7 +62,7 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
           .scrollIntoView();
 
         cy.get('figure')
-          .eq(1)
+          .eq(imageIndex)
```

Comment:
I'm not keen on relying the ordinal position of the caption-less`figure` here as needing to vary it based on the toggle will be brittle and (if this pattern) is extended make the tests harder to maintain.

However @jamesbrumpton feels this whole test (_should have a visible image without a caption that is lazyloaded and has a noscript fallback image_) should be refactored so I'm happy to go with this approach for now to unblock this issue but we should immediately look at revisiting it. We should also find a better way of selecting elements than `div div div div`. @RayNjeri please could you create an issue to refactor this test and link it here?

Reply:
Alright, totally understand.
Will go ahead and create the issue

Reply:
issue created https://github.com/bbc/simorgh/issues/4718